docs/development/pigs/pig-013.rst

```diff
@@ -15,19 +15,25 @@ PIG 13 - Gammapy dependencies and distribution
 Abstract
 ========
 
-As preparation for the planned Gammapy v1.0 release fall 2019, we propose to
-clarify which Gammapy dependencies are required and which ones are optional. We
-also propose to increase the minimum required version for some dependencies.
-Finally, we propose some updates to the Gammapy supported distribution channels
-and installation instructions.
-
-This represents that plan for 2019 to be implemented before v1.0. Things are
-likely to evolve in the coming years, as Gammapy supports different projects,
-especially CTA.
+This is a proposal to make some updates concerning Gammapy dependencies and
+supported distribution channels, to be introduced in Gammapy v0.14, and then to
+be used in Gammapy v1.0 and likely without much changes moving forward.
+
+Given that we have a good way to ship recent versions of Gammapy and all
```

Comment:
I suggest a bit of rephrasing. Gammapy is probably OK in this sense as it's still very rapidly developing, but the typical reasons for not going with latest and greatest is not that it's difficult to upgrade, but to avoid any potential upgrading issues with pipeline code (e.g upgrading gammapy might be OK, but if it also means a need to update numpy then observatories/pipelines become worried).

Reply:
I rephrased and tried to clarify a bit. But the abstract is already too long, didn't want to add much here. Gammapy isn't used in pipelines yet, and e.g. proper support for CTA pipelines and version relation to ctapipe and Gammapy is beyond the scope of this PIG. For now, both are in the prototyping phase, and ctapipe is OK to always run on the latest and greatest for now, i.e. with Gammapy we're mostly catching up here.